discovery-infra/tests/test_cancel_and_reset.py

```diff
@@ -90,3 +90,48 @@ def test_cancel_reset_after_node_boot(self, api_client, node_controller, cluster
         # self.start_cluster_install(cluster_id=cluster_id, api_client=api_client)
         # self.wait_for_nodes_to_install(cluster_id=cluster_id, api_client=api_client)
         # self.wait_for_cluster_to_install(cluster_id=cluster_id, api_client=api_client)
+
+    @pytest.mark.regression
+    def test_cancel_reset_one_node_unavailable(self, api_client, node_controller, cluster):
+        cluster_id = cluster().id
+        self.generate_and_download_image(cluster_id=cluster_id, api_client=api_client)
+        node_controller.start_all_nodes()
+        self.wait_until_hosts_are_discovered(cluster_id=cluster_id, api_client=api_client)
+        self.set_host_roles(cluster_id=cluster_id, api_client=api_client)
+        self.set_network_params(
+            cluster_id=cluster_id,
+            api_client=api_client,
+            controller=node_controller
+        )
+        self.start_cluster_install(cluster_id=cluster_id, api_client=api_client)
+        # Cancel cluster install once cluster installation start
+        self.wait_for_installing_in_progress(cluster_id=cluster_id, api_client=api_client)
+        self.cancel_cluster_install(cluster_id=cluster_id, api_client=api_client)
+        assert self.is_cluster_in_cancelled_status(
+            cluster_id=cluster_id,
+            api_client=api_client
+        )
+        # Shutdown one node
+        nodes = node_controller.list_nodes()
+        node = list(nodes.values())[0]
```

Comment:
this one depending on https://github.com/openshift/assisted-test-infra/pull/214/files
will fix it once it merged